<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordDictation</class>
 <widget class="QDialog" name="WordDictation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="again_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>发音</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="word_label">
      <property name="text">
       <string>单词：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="word_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="meaning_label">
      <property name="text">
       <string>释义：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="meaning_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="top_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="new_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>生词</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
